EventsExpress.Core/Services/AuthService.cs

```diff
@@ -134,7 +134,6 @@ public async Task ChangePasswordAsync(string oldPassword, string newPassword)
             var userDto = Context.Users
                 .Include(u => u.Account)
                     .ThenInclude(a => a.AuthLocal)
-                .AsNoTracking()
```

Comment:
I think we can just change the password here, cause validation performs all required checks for us by this moment